templates/site/node.html

```diff
@@ -29,18 +29,17 @@
             <div id="nodeMenu" class="d-flex justify-content-center node-menu-container">
                 <div id="nodeDropdownContainer" class="d-flex align-items-center h3 node-menu-item" tabindex="3">
                     Other
-                    <span class="h5 node-dropdown-chevron">
-                        <i class="fa-solid fa-chevron-down"></i>
-                    </span>
+                    <span class="h5 node-dropdown-chevron"><i class="fa-solid fa-chevron-down"></i></span>
```

Comment:
Ok this is what I mean ... there's no change here, you've just moved around the indentation and then added another empty line below. This sort of thing makes reviewing extra annoying because I now have to inspect this line to see what's changed .... and there's nothing that's changed ... and there's unnecessary spacing that we now have to clean up later.

Please be more careful and stop committing changes like these.

Reply:
Sorry, this is because this is how I like to look at the code sometimes. I'll work to not make non-functional changes from now on.